src/frontend/themes.scss

```diff
@@ -447,10 +445,14 @@ body.nord-light {
   .tour-button {
     --tour-button-color: var(--accent);
   }
-}
 
-body.nord-light .alternative > svg {
-  fill: var(--accent) !important;
+  .Sidebar .tour-button {
+    --tour-button-color: var(--primary-active);
```

Comment:
tour button in sidebar had really bad contrast

![image](https://github.com/user-attachments/assets/f95fcaf2-0a9f-4dfa-867a-cbd2e29747ec)

and now:
![image](https://github.com/user-attachments/assets/8b295f25-6d79-40b9-b810-44e38237fbfc)